### ISSUE TITLE
Please upgrade the pdfdraw server to be compatible with python3

### ISSUE THREAD

**Issue by szaimen** [strukturag/pdfdraw#24]
since python2 has nearly reached and of life, it would be great if you could support python3
Thanks!
(btw: pip isn't in ubuntu sources on ubuntu 20.04 anymore so I had to install it externally)

**Comment by lbdroid** [comment]
Its not written in python, its written in javascript. There are a few utilities written in python that are called from the javascript, but from what I can tell, they're all available for python3. I.e. "pip3 install svglib", "pip3 install PyPDF2".

The only troublesome dependency is pdftk, which seems to be dead.

**Comment by szaimen** [comment]
Yes, but because the documentation recommends the python2 libraries I thought that there might be incompatibilities with those new versions...

**Comment by lbdroid** [comment]
No, just ignore that. It does NOT depend on python2 or python at all. Just those standalone utilites that happen to be written in python.

**Comment by szaimen** [comment]
So it will work with those pyhton3 libraries as well? Did you test pdfdraw with them successfully?

**Comment by lbdroid** [comment]
Now the third time.. IT DOES NOT DEPEND ON PYTHON.

**Comment by szaimen** [comment]
Hey, please calm down!
I've understood that it doesn't depend on python. But to make all features work I've read that you definitely need to install all dependencies. That's why I thought that there might be an incompatibility with new versions of the needed libraries if the docs still recommend to use the python2 version of those.
